aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/targeting/endpointProfile/EndpointProfile.kt

```diff
@@ -45,16 +42,6 @@ internal class EndpointProfile(
     private val metrics: MutableMap<String, Double> = ConcurrentHashMap()
     private val currentNumOfAttributesAndMetrics = AtomicInteger(0)
 
-    /**
-     * Returns whether the endpoint is opted out of notification.
-     *
-     * @return String (ALL | NONE)
-     */
-    val optOut: String
```

Comment:
what is the default value for optOut? should we just pass "NONE" until we have notifications category?